addons/website_slides/models/slide_slide.py

```diff
@@ -58,9 +58,15 @@ def create(self, vals_list):
         return res
 
     def write(self, values):
+        slides_completed_change = None
```

Comment:
Could initialize to an empty browserecord so we have the same variable type in all use cases.